lib/views/pages/profile/user_info/user_account.dart

```diff
@@ -31,12 +30,14 @@ class UserAccount extends StatelessWidget {
               //TODO: get the profile info from the viewmodel
               Text(
                 userEmail ?? "averylongemailjusttocheckthatitfades",
```

Comment:
I think that from our codebase the userEmail cannot be null if the user is not null.
Moreover I think that the widget should not accept null email as input.

I think that we should replace this line by  `userEmail` and set the userEmail as type `String` and not `String?`.

In the `profile_page.dart`, once we get the asyncUserData, the email can be null in theory from the LoginUser class, but in fact it is not. So I think that we can safely replace the call to this widget to `UserAccount(userEmail: value.user.email!)`.

Reply:
Thanks for the suggestion!

The user account widget now take a String (not a String? anymore). The check for a null email is now done in profile_page directly.